### PR TITLE
Add reload-selected to IndexSheet

### PR DESCRIPTION
**Pull request by geekscrapy**
Useful to have, not only on SheetsSheet

### CODE REVIEW

File: visidata/sheets.py [hunk 1]
```diff
@@ -1052,6 +1052,7 @@ def updateColNames(sheet, rows, cols, overwrite=False):
 
 # when diving into a sheet, remove the index unless it is precious
 SheetsSheet.addCommand('g^R', 'reload-selected', 'for vs in selectedRows or rows: vs.reload()', 'reload all selected sheets')
+IndexSheet.addCommand('g^R', 'reload-selected', 'for vs in selectedRows or rows: vs.reload()', 'reload all selected sheets')
```
Comment (saulpw) [inline visidata/sheets.py:1054]: The SheetsSheet is also an IndexSheet, so we probably just want to change SheetsSheet to IndexSheet.  Please then verify that g^R still works on the SheetsSheet.

Comment (geekscrapy) [reply visidata/sheets.py:1054]: Agreed, just done this and tested with SheetsSheet 👍

**Comment by geekscrapy** [comment]
Updated to bind command to IndexSheet

**Comment by anjakefala** [comment]
Thanks @geekscrapy! =) Merged.

**Pull request merged**